runtime-testsuite/pom.xml

```diff
@@ -106,11 +109,15 @@
               <execution>
                 <phase>generate-test-sources</phase>
                 <goals>
-                  <goal>java</goal>
+                  <goal>exec</goal>
                 </goals>
                 <configuration>
-                  <mainClass>org.antlr.v4.testgen.TestGenerator</mainClass>
+                  <executable>java</executable>
                   <arguments>
+                    <argument>-Dline.separator="&#x000a;"</argument>
```

Comment:
:question: Rather than use this, why not just update `StrlenStringMap` to normalize line endings before returning the length?